File: spark/src/main/scala/com/thalys/dc/sparkcore/demo/entity/DCRecordEntity.scala
package com.thalys.dc.sparkcore.demo.entity

import com.alibaba.fastjson.JSONObject
import java.util

/**
  * descriptions:
  * DC 消息记录实体类
  *
  * author: li zhiqiang
  * date: 2020 - 12 - 21 14:51
  */
case class DCRecordEntity(ip:String, timestamp:Long, companyName:String, businessName:String, dataList:util.List[JSONObject])

File: flink/src/main/java/com/rex/demo/study/demo/entity/DCRecordEntity.java
package com.rex.demo.study.demo.entity;

import com.alibaba.fastjson.JSONObject;
import lombok.Data;

import java.util.List;

/**
 * thalys dc 数据对象
 *
 * @Author li zhiqang
 * @create 2020/11/24
 */
@Data
public class DCRecordEntity {

    private String ip;
    private Long timestamp;
    private String companyName;
    private String businessName;
    private List<JSONObject> dataList;

}